src/ci_workflow/ci_check_manifest_component.py

```diff
@@ -18,7 +18,9 @@ def __init__(self, component, url):
 
     def check(self):
         for architecture in BuildArgs.SUPPORTED_ARCHITECTURES:
-            url = "/".join([self.component.dist, architecture, "manifest.yml"])
+            # Since we only have 'linux' builds now we hard code it to 'linux'
+            # Once we have all platform builds we can adding a second loop for 'BuildArgs.SUPPORTED_PLATFORMS'
```

Comment:
Can it be `self.target.platform` instead of hard-coded `linux`?

Reply:
This `target` is `CiTarget`, which does not have the `platform` var.